app/graphql/types/conference_type.rb

```diff
@@ -20,7 +20,7 @@ def id
       object.acronym
     end
     def lectures
-      object.events
```

Comment:
@manno Any idea how to cast the array object.event from `[Events]` to `[Frontend::Event]` – I think becomes does not work for arrays...

Reply:
No idea, you probably have to do it for all elements: `events.map { |e| e.becomes(class)! }`.

But why do you need that? Maybe there is a method in Frontend that can be moved into the base class easily?

Reply:
That would mean that we move about 5-10 methods form the frontend model to the base class – if it is okay for you, I can do that..

Reply:
Well, frontend was meant for html/display related methods. If those are needed in the backend now.. You deicde.

If it's 10-20 methods, we should consider  a third layer? `Frontend < CoolNewName < Base`?